README.md

```diff
@@ -2,41 +2,70 @@
 
 ![screenshot](./screenshot.png)
 
-Patchwork is a decentralized sharing app.
-It was built for friends of the SSB project, to test basic functions.
-We use it as a test-bed for features, and for our own daily messaging.
+**Patchwork is a decentralized messaging and sharing app.**
 
-Because SSB is a research project, **you'll need to contact an SSB team member (in #scuttlebutt on Freenode) to get onto the network!**
-That's our informal barrier to entry right now, since we're not prepared for lots of users yet.
+It's better than email because:
+
+ - Private messages are end-to-end encrypted, always.
+ - You have to follow somebody to get messages from them, so you won't get spammed.
```

Comment:
The network communicates along lines of trust, so if someone breaks trust they can no longer abuse other users